packages/desktop-client/src/components/table.tsx

```diff
@@ -1239,8 +1239,8 @@ export function useTableNavigator<T extends TableItem>(
   }, []);
 
   useEffect(() => {
-    modalStackLength.current = store.getState().modals.modalStack.length;
-  }, []);
+    modalStackLength.current = modalState.modalStack.length;
```

Comment:
Unfortunately I think this approach breaks the field refocus behaviour due to caching. Steps to repro:
1. Mark transactions as reconciled
2. Edit the payee on a reconciled transaction
3. Hit 'Cancel' on the modal
On edge, this refocuses the payee field. However in this PR the field is not focused

Reply:
Should be fixed on the latest push